### PR TITLE
Feat/elasticsearch consumer

### PR DESCRIPTION
**Pull request by shawnsarwar**

### CODE REVIEW

File: elasticsearch/conf/consumer.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "elasticsearch_http_auth" : null,
+    "elasticsearch_port": 9200,
+	"connect_retry_wait": 5,
+	"startup_connection_retry": 3,
+    "latitude_fields": ["lat", "latitude"],
+	"longitude_fields": ["lon", "lng", "long", "longitude"],
+	"index_path": "/code/es_index",
+	"log_level": "DEBUG",
+	"log_name": "ES_Consumer"
+}
```
Comment (obdulia-losantos) [inline elasticsearch/conf/consumer.json]: Could you replace the tabs with spaces?

**Comment by shawnsarwar** [comment]
To test:
Wipe the kernel with `scripts/wipe_kernel.sh`
Take down connect with: `docker-compose -f ./docker-compose-connect.yml down`

Bring kernel and connect back up.
`docker-compose -f ./docker-compose.yml -f ./docker-compose-connect.yml up -d`
Generate Entities
`scripts/register_assets.sh`
`scripts/generate_assets.sh 100`
Start ES and consumer
`cd elasticsearch`
`docker-compose up`
View in Kibana @ http://localhost:5601

File: elasticsearch/es_index/aether-demo.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+			},
+			"_parent": {
+				"type": "Building"
+
+			}
+		},
+		"Survey": {}
+
+	}
+}
```
Comment (obdulia-losantos) [inline elasticsearch/es_index/aether-demo.json]: same comment as above

**Comment by obdulia-losantos** [comment]
With this line `scripts/register_assets.sh 100` I have this error message `project could not be registerd, does it already exist?` :thinking:

**Comment by obdulia-losantos** [comment]
Take a look at the typo too :stuck_out_tongue:  `registerd`

File: elasticsearch/es_index/aether-demo.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+						"Survey": "Building",
+						"Building": "Household",
+						"Household": "Person"
+					}
+				}
+			}
+
+		}
+	}
+}
```
Comment (obdulia-losantos) [inline elasticsearch/es_index/aether-demo.json]: More tabs?!!! :stuck_out_tongue:

Comment (shawnsarwar) [reply elasticsearch/es_index/aether-demo.json]: I have no idea why this happens :sob: I only use spaces!

**Pull request merged**